pulse_actions/worker.py

```diff
@@ -36,8 +38,11 @@ def run_pulse(exchanges, topics, event_handler, topic_base, dry_run):
         label = topic_base[0]
     else:
         label = str(topic_base)
-    user = os.environ.get('PULSE_USER')
-    password = os.environ.get('PULSE_PW')
+    user, password = get_user_and_password()
+
+    if user is None and password is None:
```

Comment:
I should have double-checked my code- the if statement there was left in by accident! I'll remove it and update the pull request.